### DISCUSSION TITLE
Time to remove email format option?

### DISCUSSION THREAD

**Discussion by simon1066** [zencart/zencart#5816] [Ideas]
Is it necessary to still have the customer the ability to choose the format of emails they receive - html/plain text? I can't recall ever having seen this option on websites run on other CMS'.

If we don't remove the option altogether than I think it should at least be controlled by a configuration switch. On a site of mine I have removed all references to it on signup pages but have left in in place within My Account - see how hard it is to let go, it's obviously engrained into my subconscious!

I appreciate this is a bit late in the day for ZC v1.5.8 but perhaps a subsequent release. Before anyone asks I don't think I have the [coding] skills to submit a PR for this.

**Comment by brittainmark** [comment]
I don't give people the option and force all to plain text. it's more secure and less likely to end up in spam.

**Comment by dbltoe** [comment]
I have to disagree here.  With SPF, DKIM, and especially DMARC properly set, your likelihood of being sent to spam is nil.
With plain text, your likelihood of sending a pleasing, professional email is also nil.
